bin/github-backup

```diff
@@ -868,6 +875,26 @@ def backup_account(args, output_directory):
                      output_file,
                      account_cwd)
 
+    if args.include_followers or args.include_everything:
+        output_file = '{0}/followers.json'.format(account_cwd)
+        template = "https://{0}/users/{1}/followers"
```

Comment:
missing call to `.format()` here? also using double-quotes instead of single quotes like the rest of the file